src/main/java/com/chessmates/repository/RdsQueryExecutor.groovy

```diff
@@ -103,14 +103,10 @@ class RdsQueryExecutor implements QueryExecutor {
                 outputObject
         }
 
-        //lambda placeholder
-        def populateNone = {
 
-        }
-
-        //we only populate if it was a select query inwhich case it returns a result set
-        def returnValue = item instanceof ResultSet ? populateOutputObject(item) : populateNone
+        //we only populate if it was a select query in which case it returns a result set
+        if(!(item instanceof ResultSet)) return
```

Comment:
In this case it doesn't really matter what is returned. This 'return' will only execute for Insert queries and we are not doing anything with the return value for inserts.